addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -782,11 +784,11 @@ export function getCursors(document) {
 export function preserveCursor(document) {
     const sel = document.getSelection();
     const cursorPos = [sel.anchorNode, sel.anchorOffset, sel.focusNode, sel.focusOffset];
-    return replace => {
+    return (replace, normalize) => {
```

Comment:
I suggest using an options object otherwise we have to pass `undefined` just to skip the first parameter which is sad.